Sources/StreamChatUI/ChatMessageList/ChatMessageListVC+SwiftUI.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+@available(iOS 13.0, *)
+/// A `UIViewControllerRepresentable` subclass which wraps `ChatMessageListVC` and shows list of channels.
+public typealias ChatMessageListView = _ChatMessageListVC<NoExtraData>.View
+
+@available(iOS 13.0, *)
+extension _ChatMessageListVC {
+    /// A `UIViewControllerRepresentable` subclass which wraps `ChatMessageListVC` and shows list of channels.
+    public struct View: UIViewControllerRepresentable {
```

Comment:
nit: `UIViewControllerRepresentable` is a protocol so it can't have subclasses :) Maybe `A SwiftUI View that wraps ...` ?

Reply:
Thanks for the spot, fixed it both here and where I copied this from 😬